File: app/src/main/java/com/android/mrt/mvp/java/activity/MainActivity.java
package com.android.mrt.mvp.java.activity;

import android.widget.TextView;

import com.android.mrt.R;
import com.android.mrt.common.base.BaseMvpActivity;
import com.android.mrt.mvp.java.contract.IMainContract;
import com.android.mrt.mvp.java.presenter.MainPresenter;

public class MainActivity extends BaseMvpActivity<IMainContract.IMainView, MainPresenter> implements IMainContract.IMainView {

    @Override
    protected int getMContentViewRes() {
        return R.layout.activity_main;
    }

    @Override
    protected void initView() {

    }

    @Override
    protected void initData() {
        getPresenter().getTitle();
        getPresenter().getUser();
    }

    @Override
    public void showTitle(String title) {
        ((TextView) findViewById(R.id.mTitle_TV)).setText(title);
    }

    @Override
    public void showUser(String user) {
        ((TextView) findViewById(R.id.mUser_TV)).setText(user);
    }
}

File: app/src/main/java/com/android/mrt/mvp/kotlin/activity/MainActivity.kt
package com.android.mrt.mvp.kotlin.activity

import com.android.mrt.R
import com.android.mrt.common.base.BaseMvpActivity
import com.android.mrt.mvp.kotlin.contract.IMainContract
import com.android.mrt.mvp.kotlin.presenter.MainPresenter
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : BaseMvpActivity<IMainContract.IMainView, MainPresenter>(), IMainContract.IMainView {

    override val mContentViewRes: Int = R.layout.activity_main

    override fun initView() {

    }

    override fun initData() {
        getPresenter()?.getTitle()
        getPresenter()?.getUser()
    }

    override fun showTitle(title: String) {
        mTitle_TV.text = title
    }

    override fun showUser(user: String) {
        mUser_TV.text = user
    }
}